### ISSUE TITLE
Cannot provide a folder with my csv files for import in REST API

### ISSUE THREAD

**Issue by lhubert** [Hurence/historian#132]
Seems that the REST API is only allowing for a list of files to be given in :
curl -X POST \
http://localhost:8080/historian-server/ingestion/csv \
-F 'my_csv_file=@path/de/mon/csv/file.csv' \
-F 'my_csv_file2=@path/de/mon/csv/file2.csv' \
-F mapping.name=metric_name_2 \
-F mapping.value=value_2 \
-F mapping.timestamp=timestamp \
-F mapping.quality=quality \
-F mapping.tags=sensor \
-F mapping.tags=code_install \
-F timestamp_unit=MILLISECONDS_EPOCH \
-F group_by=name \
-F group_by=tags.sensor \
-F format_date=yyyy-D-m HH:mm:ss.SSS \
-F timezone_date=UTC \

**We need to provide a path to a folder** instead of having to list all files contained in a folder.

**Comment by MiniPlayer** [comment]
This can be a nice feature. But this endpoint is not meant to handle high csv volume. It is designed to handle small files otherwise the best choice is to use the spark api.

**Comment by lhubert** [comment]
if you read files one by one I do not see the problem. You will create your chunks on a per file basis... still... and in your endpoint you will have the same code just applied n times

**Comment by MiniPlayer** [comment]
Yes indeed. But his means we may have to provide a mechanism for long request to proceed (if folder contain 2000 files...). In this case we must respond that the request has beenr eceived and is being processed and provide a way for the user to consult the status of the process. 
This is just informative for when we implement this.
